content/docs/introduction/pro-plan.md

```diff
@@ -34,6 +34,7 @@ The Neon Pro Plan comes with the following advanced features, and we plan to add
 - **Configurable Auto-suspend:** The Neon Free Tier automatically suspends computes after 5 minutes of inactivity. With the Neon Neon Pro Plan, you can increase or decrease this time period, or you can disable the Auto-suspend feature entirely. A shorter delay, such as 60 seconds, can potentially reduce compute costs. A longer delay keeps your compute active for longer intervals, reducing the number of cold starts. To prevent cold starts entirely, in cases where even a few seconds delay is too much, you can disable the Auto-suspend feature entirely so that your compute is always active. The Auto-suspend feature is configured in the Neon Console. For instructions, see [Auto-suspend configuration](/docs/manage/endpoints#auto-suspend-configuration).
 - **Project sharing:** The _Project sharing_ feature lets you share your Neon projects with other Neon users. Project sharing is managed on the **Settings** page in the Neon Console. Simply add the email account of the user you want to share your project with. Projects can be shared with any Neon user, including Neon Free Tier users. Usage is applied to the project owner's Neon account. For instructions, see [Share a project](/docs/manage/projects#share-a-project).
 - **Read replicas:** Neon's read replica feature enhances scalability and performance by allowing you to create read-only compute instances that operate on the same data as your read-write computes. With this feature, you can offload analytics or reporting queries, efficiently distribute read requests to achieve higher throughput, or provide read-only data access to certain users or applications. For more information, see [Read replicas](/docs/introduction/read-replicas).
+- **IP Allowlist:** Control access to the branch where your database resides. Any IP address not on the allowlist is automatically blocked, ensuring that only authorized users can access your data. The allowlist can be applied to all branches (the default) or to your [primary branch](#primary-branch) only. For more information, see [IP Allowlist](/docs/manage/projects#configure-the-ip-allowlist).
```

Comment:
Out of curiosity, why would you only apply it to the primary branch, but not other branches?